data/mods/Magiclysm/worldgen/goblin_encampment.json

```diff
@@ -0,0 +1,578 @@
+      "goblin_4B_roof",
+      "goblin_5B_roof",
+      "goblin_1C_roof",
+      "goblin_2C_roof",
+      "goblin_3C_roof",
+      "goblin_4C_roof",
+      "goblin_5C_roof"
+    ],
+    "name": "goblin encampment",
+    "sym": "#",
+    "color": "red",
+    "see_cost": 5
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [
+      [ "goblin_1A_roof", "goblin_2A_roof", "goblin_3A_roof", "goblin_4A_roof", "goblin_5A_roof" ],
+      [ "goblin_1B_roof", "goblin_2B_roof", "goblin_3B_roof", "goblin_4B_roof", "goblin_5B_roof" ],
+      [ "goblin_1C_roof", "goblin_2C_roof", "goblin_3C_roof", "goblin_4C_roof", "goblin_5C_roof" ]
+    ],
+    "object": {
+      "fill_ter": "t_shingle_flat_roof",
+      "rows": [
+        "                                                                                                                        ",
+        "                                                                                                                        ",
+        "                                                                                                                        ",
+        "                                                                                                                        ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .....................................           ....................................................           ",
+        "         .......>                                                    >                                .......           ",
+        "         .......                                                                                      .......           ",
+        "         .......                                                                                      .......           ",
+        "         .......                                                                                      .......           ",
+        "         .......                                                                                      .......           ",
+        "         .......         *                       *                       *                       *    .......           ",
```

Comment:
Looks like `*` is not defined in mapgen palette.

Reply:
these were map boundaries I forgot to remove so they can be deleted